egs2/aishell4/asr1/conf/tuning/train_asr_conformer5.yaml

```diff
@@ -0,0 +1,78 @@
+# network architecture
```

Comment:
It's good to have different configurations, but we could just keep the best conf for the initial recipe.